tests/manage/z_cluster/test_rook_ceph_osd_flapping.py

```diff
@@ -0,0 +1,104 @@
+import logging
```

Comment:
Please move this test to the new path /tests/functional/z_cluster

Reply:
done